guides/common/modules/con_smart-proxy-configuration-tuning.adoc

```diff
@@ -4,4 +4,10 @@
 = {SmartProxy} configuration tuning
 
 {SmartProxies} are meant to offload part of {Project} load and provide access to different networks related to distributing content to clients but they can also be used to execute remote execution jobs.
-What they cannot help with is anything which extensively uses {Project} API as host registration or package profile update.
+What they cannot help with is anything which extensively uses {Project} API,
+ifdef::katello,orcharhino,satellite[]
+for example, host registration or package profile update.
+endif::[]
+ifdef::foreman-el,foreman-deb[]
+for example, host registration.
+endif::[]
```

Comment:
```suggestion
What they cannot help with is anything which extensively uses {Project} API,
ifdef::katello,orcharhino,satellite[]
such as host registration or package profile update.
endif::[]
ifdef::foreman-el,foreman-deb[]
such as host registration.
endif::[]
```
To reduce the number of commas because they made the result a bit difficult to parse.